lib/rails/sharding.rb

```diff
@@ -1,4 +1,7 @@
+require 'active_record'
+require 'active_support/core_ext'
 require 'rails'
```

Comment:
This is not the full Rails gem, it's only `railties`.

Reply:
Not sure I'm following... the `require 'rails'` loads only the railties when you don't have the rails gem installed? Is that it?

Reply:
That is correct. Instead of using the name `Railties`, they use `Rails`, however, the `rails` gem is not included.